apps/dashboard/app/javascript/packs/script_edit.js

```diff
@@ -121,10 +121,33 @@ function addInProgressField(event) {
   justAdded.find('[data-select-toggler]')
            .on('click', (event) => { enableOrDisableSelectOption(event) });
 
+  justAdded.find('[data-fixed-toggler]')
+           .on('click', (event) => { toggleFixed(event) });
+
   const entireDiv = event.target.parentElement.parentElement.parentElement;
   entireDiv.remove();
 }
 
+function fixedFieldEnabled(checkbox, dataElement) {
+  dataElement.disabled = true;
+  const input = $('<input>').attr('type','hidden').attr('name', dataElement.name).attr('value', dataElement.value);
+  $(checkbox).after(input);
+}
+
+function toggleFixed(event) {
+  event.target.disabled = true;
```

Comment:
This is usually a good practice to avoid double submitting/processing while the first click is being handled.

In this particular case might too much. I can remove if you think it is not needed.

Reply:
> This is usually a good practice to avoid double submitting/processing while the first click is being handled.

First click? Are you saying this event is triggered twice by 1 click (click up and click down events?) or do you mean double clicking?

We can keep it if it serves some purpose, as you say as a good practice, I'm just curious mostly now.

Reply:
I meant double clicking. But I think in this particular case even a double click will have no side effects.